projects/ima-namespace/ima-utils/Makefile

```diff
@@ -7,6 +7,7 @@ DEPS=Dockerfile
 HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
 
 tag: $(DEPS)
+	docker pull linuxkit/$(IMAGE):$(HASH) || \
```

Comment:
all the Makefiles now have:
```
ORG?=linuxkit
```
and then
```
push: tag
        docker pull $(ORG)/$(IMAGE):$(HASH) || \
        docker push $(ORG)/$(IMAGE):$(HASH)
```
allows you to override to push to private registry or hub org